equinity-historical-data/src/main/java/uk/gov/justice/laa/crime/equinity/historicaldata/mapper/Crm14Mapper.java

```diff
@@ -609,6 +626,41 @@ default String assignFileKey(Crm14AttachmentModel processAttachModel) {
         }
         return null;
     }
+    @Named("hasCrm15")
+    default boolean hasCrm15(Crm14DetailsModel model) {
```

Comment:
I think this function should be moved to class Crm14DetailsModel. That will encapsulate that logic which relies only on the model object itself. It can also make testing far easier. Besides, in my opinion, knowing if it has a 15 or not is not part of the mapper but of the model.
Happy to do this in another commit. But add a TOTO so we do not forget to do it at some point

Reply:
Thanks @tanovellino  Moved Mapper code to Model class.